hieroplatform/src/main/java/org/hiero/sketches/SampleCorrelationSketch.java

```diff
@@ -0,0 +1,88 @@
+package org.hiero.sketches;
+
+import org.hiero.dataset.api.ISketch;
+import org.hiero.table.api.ContentsKind;
+import org.hiero.table.api.IMembershipSet;
+import org.hiero.table.api.IRowIterator;
+import org.hiero.table.api.ITable;
+
+import javax.annotation.Nullable;
+import java.security.InvalidParameterException;
+import java.util.List;
+
+/**
+ * A sketch to compute correlations between columns, using sampling.
+ */
+public class SampleCorrelationSketch implements ISketch<ITable, CorrMatrix>{
+    /**
+     * The list of columns whose correlations we wish to compute. The columns are must be of type
+     * Int or Double.
+     */
+    private final List<String> colNames;
+    /**
+     * The probability of a row being included in the sample, default value is 0.1
```

Comment:
Maybe the default value comment should be applied to the constructor below; if you use the first constructor this does not apply.